src/main/java/uk/gov/hmcts/cmc/claimstore/controllers/CountyCourtJudgmentController.java

```diff
@@ -19,6 +19,8 @@
 import javax.validation.Valid;
 import javax.validation.constraints.NotNull;
 
+import static uk.gov.hmcts.cmc.claimstore.controllers.ClaimController.UUID_PATTERN;
```

Comment:
That does not sound right, why this controller would depend on the other one? Pattern should get extracted to separate class.

Reply:
done